pkg/tagset/default.go

```diff
@@ -0,0 +1,55 @@
+package tagset
+
+// DefaultFactory is a global thread-safe factory, used by calls to
+// package-level functions.  This is suitable for non-performance-critical tags
+// manipulation
+var DefaultFactory Factory
```

Comment:
Do we need to export the `DefaultFactory` or would it be enough to just export the functions below?

Reply:
I suspect there are some cases where it's useful to use the DefaultFactory.  This is similar to `net/http`'s `DefaultServeMux`, for example.